libbeat/beat/beat.go

```diff
@@ -213,10 +216,19 @@ func (b *Beat) LoadConfig() error {
 		return fmt.Errorf("error Initialising publisher: %v\n", err)
 	}
 
+	filters, err := filter.New(b.Config.Filter)
+	if err != nil {
+		fmt.Printf("Error Initialising filters: %v\n", err)
+		logp.Critical(err.Error())
```

Comment:
I noticed we spell "initializing" two different ways in messages in this file. We should stop choose one way and change them all.